src/ApplicationAspectKernel.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace KickAss\Commerce;
+
+use Go\Core\AspectKernel;
+use Go\Core\AspectContainer;
+
+/**
+ * Application Aspect Kernel
+ */
+class ApplicationAspectKernel extends AspectKernel
+{
+
+    /**
+     * Configure an AspectContainer with advisors, aspects and pointcuts
+     *
+     * @param AspectContainer $container
+     * @return void
+     */
+    protected function configureAop(AspectContainer $container)
```

Comment:
dude.... as I said, I just copy pasted a bunch of code and it worked.

But it can prob be public...